README.md

```diff
@@ -207,12 +211,21 @@ API | Description | Auth | HTTPS | Link |
 | RandomUser | Generates random user data | No | Yes | [Go!](https://randomuser.me) |
 | RandomWord | Generate random word | No | No | [Go!](http://www.setgetgo.com/randomword/) |
 | ReqRes | A hosted REST-API ready to respond to your AJAX requests | No | Yes | [Go!](https://reqres.in/ ) |
-| Restpack | Provides screenshot, HTML to PDF, and content extraction APIs | `apiKey` | Yes | [Go!](https://restpack.io/) |
 | RoboHash | Generate random robot/alien avatars | No | Yes | [Go!](https://robohash.org/) |
+| Scrape Website Email | Grabs email addresses from a URL | `X-Mashape-Key` | Yes | [Go!](https://market.mashape.com/tommytcchan/scrape-website-email) |
+| SHOUTCLOUD | ALL-CAPS AS A SERVICE | No | No | [Go!](http://shoutcloud.io/) |
 | StackExchange | Q&A forum for developers | `OAuth` | Yes | [Go!](https://api.stackexchange.com/) |
 | UI Names | Generate random fake names | No | Yes | [Go!](https://github.com/thm/uinames) |
 | Verse | Check what's the latest version of your favorite open-source project | No | Yes | [Go!](https://verse.pawelad.xyz/) |
 
+### Dictionaries
+API | Description | Auth | HTTPS | Link |
+|---|---|---|---|---|
+| Oxford | Dictionary Data | `apiKey` | Yes | [Go!](https://developer.oxforddictionaries.com/) |
+| Pearson | Dictionary Data | `apiKey` | No | [Go!](http://developer.pearson.com/apis/dictionaries) |
```

Comment:
`The Dictionary API beta program has been suspended and the API is in a deprecated status. The API is still currently accessible to existing applications although public availability will be ending before the end of 2017. A notification will be sent to active developers once an exact shutdown schedule has been determined.`

Deprecated service, please remove

Reply:
Ah, that service was in the list before I made this edit, but it was in the `Development` section, so I didn't actually check it out. I'll remove it.